numba/cuda/cudadecl.py

```diff
@@ -799,3 +800,7 @@ def resolve_local(self, mod):
 
 for func in bit_twiddling_functions:
     register_numpy_ufunc(func, register_global)
+
+for func in math_operations:
+    if func in ('log2', 'log', 'log10'):
```

Comment:
Would being specific by using something like the following be preferable to iterating over all the math operations and masking off those supported?
```python
for name in ('log2', 'log', 'log10'):
  func = getattr(math_operations, name, None)
  if func is None:
    raise # something appropriate 
  ... # do registration
```

Reply:
Sounds good. But just a remainder that `math_operations` is actually a list containing the name of supported math functions, not the math module itself.

```python
(Pdb++) math_operations
['add', 'subtract', 'multiply', 'logaddexp', 'logaddexp2', 'true_divide', 'floor_divide', 'negative', 'positive', 'power', 'float_power', 'remainder', 'fmod', 'absolute', 'rint', 'sign', 'conjugate', 'exp', 'exp2', 'log', 'log2', 'log10', 'expm1', 'log1p', 'sqrt', 'square', 'cbrt', 'reciprocal', 'divide', 'mod', 'divmod', 'abs', 'fabs', 'gcd', 'lcm']
```